linkerd/examples/acceptance-test.yaml

```diff
@@ -49,6 +49,11 @@ routers:
     httpUriInDst: true
   responseClassifier:
     kind: retryableIdempotent5XX
+  announcers:
+  - kind: io.l5d.announcer.Serversets
```

Comment:
i'll comment on your proposal, but I don't think we should have "announcer" in this -- we're in an announcers block, what would it mean to specify anything else?  Also, i'd prefer that we avoid capitalization.